mage_ai/frontend/components/datasets/FeatureProfiles.tsx

```diff
@@ -88,95 +115,73 @@ function FeatureProfile({
-        'Minimum',
-        'Maximum',
-        'Invalid',
-      ],
-      [
-        columnType === ColumnTypeEnum.DATETIME ? medianValue : roundNumber(meanValue, 3),
-        typeof minValue === 'string' ? minValue: roundNumber(minValue, 3),
-        typeof maxValue === 'string' ? maxValue: roundNumber(maxValue, 3),
-        numberOfInvalidValues,
-      ],
-    ];
-  } else {
-    entries = [
-      [
-        'Type',
-        'Count',
-        'Unique',
-      ],
-      [
-        COLUMN_TYPE_HUMAN_READABLE_MAPPING[columnType] || columnType,
-        numberOfValues,
-        numberOfUniqueValues,
-      ],
-      [
-        'Missing',
-        'Mode',
-        'Invalid',
-      ],
-      [
-        numberOfNullValues,
-        modeValue,
-        numberOfInvalidValues,
-      ],
-    ];
-  }
+  const entries = [
+    columnType,
+    numberOfUniqueValues,
+    meanValue,
+    medianValue,
+    modeValue,
+    minValue,
+    maxValue,
+    numberOfNullValues,
+    numberOfInvalidValues,
+    numberOfOutliers,
+    skewness,
+    stddev,
+  ];
 
   const featureSetId = featureSet.id;
   const featureUuid = feature?.uuid;
-  const featureId = getFeatureIdMapping(featureSet)[featureUuid]
+  const featureId = getFeatureIdMapping(featureSet)[featureUuid];
 
   return (
-    <FlexContainer>
-      <Flex flex={1}>
-        <CellStyle>
+    <Flex flexDirection="column">
+      <FeatureProfileStyle>
+        <Spacing p={2}>
           <NextLink
             as={`/datasets/${featureSetId}/features/${featureId}`}
             href="/datasets/[...slug]"
             key={featureUuid}
             passHref
           >
             <Link inline>
-              <Text backgroundColor={PURPLE_HIGHLIGHT} bold color={PURPLE} monospace>
+              <Text
+                backgroundColor={PURPLE_HIGHLIGHT}
+                bold
+                color={PURPLE}
+                maxWidth={25 * UNIT}
+                monospace
+                textOverflow
+                title={uuid}
+              >
                 {uuid}
               </Text>
             </Link>
           </NextLink>
-        </CellStyle>
-      </Flex>
-      {entries.map((values, idx) => (
-        <Flex flex={1} flexDirection="column" key={`column-${idx}`}>
-          {values.map((label, idx) => (
-            <CellStyle backgroundColor={idx % 2 === 0 ? WHITE : LIGHT} key={idx}>
-              <Text>
-                {label}
-              </Text>
-            </CellStyle>
-          ))}
-        </Flex>
-      ))}
-    </FlexContainer>
+        </Spacing>
+      </FeatureProfileStyle>
+      {entries.map((label = '-', idx) => {
+        const entry = entryTypes[idx];
+        const val = !isNaN(label) ? roundNumber(label) : label;
+        const shouldWarn = entry in warnings && (val/numberOfValues) > warnings[entry];
+
+        return (
+          <CellStyle backgroundColor={idx % 2 === 0 ? WHITE : LIGHT} key={idx}>
+            <Text
+              color={shouldWarn && 'red'}
```

Comment:
Instead of manually setting the color, use the Text prop that uses the color from the theme. Prop on Text might be "danger".